themes/ddbasic/sass/components/ui-datepicker.scss

```diff
@@ -0,0 +1,98 @@
+@import '../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?